oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -426,6 +428,20 @@ private void checkIfStopped() throws CommitFailedException {
 
     @Override
     public synchronized void run() {
+        boolean isIndexWithLanePresent = false;
+        NodeState oakIndexNode = store.getRoot().getChildNode("oak:index");
+        Iterator<? extends ChildNodeEntry> indexNodeIterator = oakIndexNode.getChildNodeEntries().iterator();
+        while (indexNodeIterator.hasNext()){
+            ChildNodeEntry  childNodeEntry = indexNodeIterator.next();
```

Comment:
minor: this can be made a bit more concise with

```suggestion
        for (ChildNodeEntry childNodeEntry : oakIndexNode.getChildNodeEntries()) {
```